base/bucket_n1ql_test.go

```diff
@@ -575,3 +574,37 @@ func tearDownTestIndex(bucket *CouchbaseBucketGoCB, indexName string) (err error
 	}
 
 }
+
+func TestWaitForBucketExistence(t *testing.T) {
+	if UnitTestUrlIsWalrus() {
+		t.Skip("This test only works against Couchbase Server")
+	}
+
+	testBucket := GetTestBucket(t)
+	defer testBucket.Close()
+	bucket, ok := testBucket.Bucket.(*CouchbaseBucketGoCB)
+	assert.True(t, ok, "Requires gocb bucket")
```

Comment:
Stop the test when this isn't true, we can't continue.

```suggestion
	require.True(t, ok, "Requires gocb bucket")
```

Reply:
Makes more sense; this is fixed.